solutions/cf-831C.mdx

```diff
@@ -50,5 +50,52 @@ int main() {
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class JuryMarks {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio();
+		int numJury = io.nextInt();
+		int numScores = io.nextInt();
+		int[] marks = new int[numJury + 1];
+		int[] scores = new int[numScores];
+
+		for (int x = 1; x <= numJury; x++) {  // read k numbers
+			marks[x] = io.nextInt() + marks[x - 1];
+		}
+		for (int x = 0; x < numScores; x++) {  // read n numbers
+			scores[x] = io.nextInt();
+		}
+		Set<Integer> init = new TreeSet<>();
+		for (int x = 1; x <= numJury; x++) { // add to set
+			init.add(scores[0] - marks[x]);
+		}
+		int answer = 0;
```

Comment:
uh don't think renaming `ans` to `answer` was necessary lmao i'm not that nitpicky

Reply:
oh i probably did that unintentionally. but you are one of the strictest reviewers though